frontend/jupyter/src/app/pages/index/index-default/index-default.component.ts

```diff
@@ -356,10 +360,6 @@ export class IndexDefaultComponent implements OnInit, OnDestroy {
     return `${notebook.name}/${notebook.image}`;
   }
 
-  private updateButtons(): void {
-    this.buttons = [this.newNotebookButton];
-  }
-
```

Comment:
Assuming this is going to get removed from https://github.com/StatCan/jupyter-apis/blob/ae0be6334f1040e5e99c12745161e55051bc4b49/frontend/jupyter/src/app/pages/notebook-page/notebook-page.component.spec.ts#L160 when tests are getting fixed. Might want to add a comment

Reply:
Irrelevant. That function is for index-default. 

You linked the test file for notebook-page. That updateButtons function refers to this one
https://github.com/StatCan/jupyter-apis/blob/ae0be6334f1040e5e99c12745161e55051bc4b49/frontend/jupyter/src/app/pages/notebook-page/notebook-page.component.ts#L207

Different function entirely, even if they shared a name